### DISCUSSION TITLE
Lists for star repos

### DISCUSSION THREAD

**Discussion by Shoaib19** [community/community#106377] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am creating a list for my star repos it created but why does UI get stuck?

https://github.com/community/community/assets/86943632/f6904fde-c741-48c1-8ae1-aa0b16fb7559

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Shoaib19! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by Shoaib19** [reply]
Thanks for moving it to right category.

**Reply by Isa232323** [reply]
- [ ]
